src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -173,4 +174,5 @@ function ProductGallery({ title, searchTerm }: Props) {
   )
 }
 
-export default ProductGallery
+ProductGallery.displayName = 'ProductGallery'
```

Comment:
Is it worth adding a test in `performance.test.js` to assert it's only rendered once for the PLP?

Reply:
Nice!